test/e2e/apimachinery/crd_conversion_webhook.go

```diff
@@ -382,6 +382,8 @@ func testCRListConversion(f *framework.Framework, testCrd *framework.TestCrd) {
 	Expect(len(list.Items)).To(BeIdenticalTo(2))
 	Expect((list.Items[0].GetName() == name1 && list.Items[1].GetName() == name2) ||
 		(list.Items[0].GetName() == name2 && list.Items[1].GetName() == name1)).To(BeTrue())
+	// Make sure the conversion happened in one iteration
+	Expect(list.Items[0].Object["_debug_conversion_iteration_id"] != nil && list.Items[0].Object["_debug_conversion_iteration_id"] == list.Items[1].Object["_debug_conversion_iteration_id"])
```

Comment:
I'm actually surprised both objects were sent to the webhook... I thought we only sent objects needing conversion, and in this case, only one item should have required conversion, right? (the object stored in v1 wouldn't need conversion when fetching the v1 list, and the object stored in v2 wouldn't need conversion when fetching the v2 list)

Reply:
I see, right. It could be a bug, I will follow it and send a bugfix pr.